arrowhead/arrowhead.rb

```
@@ -15,19 +20,14 @@ class Arrowhead
     },
   }
 
-  # FIXME: I don't have time to deal with this.
   def self.classify(region, shape)
-    if CLASSIFICATIONS.include? region
-      shapes = CLASSIFICATIONS[region]
-      if shapes.include? shape
-        arrowhead = shapes[shape]
-        "You have a(n) '#{arrowhead}' arrowhead. Probably priceless."
-      else
-        raise "Unknown shape value. Are you sure you know what you're talking about?"
-      end
-    else
-      raise "Unknown region, please provide a valid region."
-    end
+    region_shapes = CLASSIFICATIONS[region]
+    arwhead = region_shapes.nil? ? raise(UNKNOWN_REGION) : region_shapes[shape]
```

Avoid ternary operators, especially when determining if you need to raise an error.  Ternary operators should only be used for the _most basic_ of expressions.